contracts/VeJoeStaking.sol

```diff
@@ -60,6 +63,9 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
     /// @notice The length of time a user receives boosted benefits
     uint256 public boostedDuration;
 
+    /// @notice The upper limit of `boostedDuration`
+    uint256 public UPPER_LIMIT_BOOSTED_DURATION;
```

Comment:
One of the feedback I have received before is not to name these variables in UPPERCASE_WITH_UNDERSCORES as they aren't actually `immutable` variables.